spec/rubocop/cop/metrics/abc_size_spec.rb

```diff
@@ -39,7 +39,7 @@ def method_name
     it 'registers an offense for an assignment of an element' do
       expect_offense(<<~RUBY)
         def method_name
-        ^^^^^^^^^^^^^^^ Assignment Branch Condition size for method_name is too high. [<1, 1, 0> 1.41/0]
+        ^^^^^^^^^^^^^^^ Assignment Branch Condition size for method_name is too high. [<1, 2, 0> 2.24/0]
```

Comment:
The idea is that since `x` is not a variable, `x` is a method call (+1 branch). Then `.[]=` is another method call, so counts as +1 for method call and +1 for assignment.

Reply:
I forgot that `x` is not a variable. You're right, it should be `<1, 2, 0>`.